### DISCUSSION TITLE
Force using C2C'S re-recorded voices for 2SP / Allow selecting girl's voices freely in any mode

### DISCUSSION THREAD

**Discussion by Sonic-TV** [emoose/OutRun2006Tweaks#264] [Suggestions]
For example, force using C2C Clarissa's voice for 2SP rather than the original one.

**Comment by Sonic-TV** [comment]
> You mean the voice that shouts "OutRun Mode!", "Automatic Transmission!", "Magical Sound Shower!" and things like that?
> 
> This reminds me of issue #14 where the announcer is mute in the OutRun2SP selection screens when using foreign languages for whatever reason.

Not the menu selection as these are not present in C2C.
It's the "Checkpoint" sound and "I wanna go far away“ etc.
Also, the Mission Information for heart attack mode. (i.e. Keep passing cars!)

**Comment by Tenome** [comment]
That'd be an interesting feature. For now I guess we can swap out the audio files and have it as a mod.
